source/options.html

```diff
@@ -1,33 +1,10 @@
 <!doctype html>
 <meta charset="utf-8">
 <title>Refined GitHub options</title>
-<style>
-	body {
-		width: 350px;
-		margin: 14px 17px; /* Chrome default, match for Firefox */
-		font-family: sans-serif;
```

Comment:
These properties no longer apply to Chrome, hence the slightly improved/different style there.